### PR TITLE
Adds fftjs and remove jsfft-for-meyda

### PR DESCRIPTION
**Pull request by nevosegal**
This branch removes the external fft library we've been using and instead uses our very own fft library - [fftjs](https://github.com/nevosegal/fftjs) :1st_place_medal:

### CODE REVIEW

**Comment by 2xAA** [comment]
Is there any change in functionality?
I can't get the node or web `dist`s to work within modV.

Keep getting `NaN` in amplitudeSpectrum and complexSpectrum :(

screen shot 2017-04-23 at 14 36 38

**Comment by 2xAA** [comment]
CLI also produces this - is testing broken?

screen shot 2017-04-23 at 14 46 31

**Comment by nevosegal** [comment]
OK yeah that's a problem in the library. I'll fix and push. I don't think the tests are broken, we just don't check for multiple spectrums in a row, and that's where it failed.

**Comment by nevosegal** [comment]
OK, that should be fixed now. @2xAA if you can check how well it works now with modV that would be great. I haven't checked whether it's quicker/slower than the other fft library, but it should work very well in real time.

**Comment by 2xAA** [comment]
Alright, working nicely with modV and CLI is functioning well also.
Seems to be just as quick in realtime to me, no noticeable difference anyway :)

Nice work @nevosegal !

File: package.json [hunk 1]
```diff
@@ -70,8 +70,8 @@
   },
   "dependencies": {
     "dct": "0.0.3",
+    "fftjs": "0.0.2-alpha.1",
```
Comment (hughrawlinson) [inline package.json]: I'm glad it works - why don't you do a non-alpha release on fftjs and then upgrade to that here. Then we can merge! :)

**Pull request merged**